.github/workflows/c-cpp_tmpl.yml

```diff
@@ -31,6 +31,29 @@ jobs:
         uses: NWChemEx-Project/.github/actions/get_dependencies@master
         with:
           dependencies: ${{inputs.dependencies}}
+      - name: cache-libint
+        if: github.event.repository.name == 'NWChemEx'
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: install
+          key: libint-gcc${{ matrix.gcc_version }}
+      - name: Build libint
+        if: |
+          github.event.repository.name == 'NWChemEx' &&
+          steps.cache.outputs.cache-hit != 'true'
+        run: |
+          export INSTALL_PATH=`pwd`/install
+          # check code out
+          wget https://github.com/evaleev/libint/releases/download/v2.6.0/libint-2.6.0.tgz
+          tar -zxf libint-2.6.0.tgz
+          cd libint-2.6.0
+          export CXX=`which g++`
+          export CC=`which gcc`
+          ../cmake-3.16.3-Linux-x86_64/bin/cmake -H. -Bbuild -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_FLAGS="-std=c++17" -DBUILD_SHARED_LIBS=ON -DCPP_GITHUB_TOKEN=$CPP_GITHUB_TOKEN
+          cd build
+          make
+          make install
```

Comment:
Can we move this into the script which gets dependencies? Libint is a dependency of at least two repos.

Reply:
done! please check now.